mcs/class/corlib/Test/System.Reflection.Emit/ModuleBuilderTest.cs

```diff
@@ -794,5 +794,18 @@ public void GetConstructorTokenNullParam ()
 			// ArgumentNullException should not occur.
 			module.GetConstructorToken (method, null);
 		}
+
+		[Test]
+		public void GetType ()
+		{
+			AssemblyBuilder ab = genAssembly ();
+			ModuleBuilder module = ab.DefineDynamicModule ("foo.dll", "foo.dll", true);
```

Comment:
@vargaz is the emitSymbolInfo=true overload necessary for the test here? It fails on Android because a symbol writer is not available.